### PR TITLE
Fix/bugs

### PR DESCRIPTION
**Pull request by Supergamer1337**
## Fixes implemented
- Fixed issue with creating new accounts on the current build.
- Added workaround for ProcessRequest not working with Express V5.
- Removed very annoying error printing whenever someone logs in.
- Const.

### CODE REVIEW

File: backend/src/middleware/exceptionLoggingMiddleware.ts [hunk 1]
```diff
@@ -7,6 +7,13 @@ export const logExceptions = (
 	res: Response,
 	next: NextFunction
 ) => {
-	winston.error(err.message, { stacktrace: err.stack });
+	debugger;
```
Comment (Krysset) [inline backend/src/middleware/exceptionLoggingMiddleware.ts:10]: I don't understand this line 😅

Comment (Supergamer1337) [reply backend/src/middleware/exceptionLoggingMiddleware.ts:10]: Oj

File: frontend/src/pages/callback.tsx [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
+			redirect: {
+				destination: '/',
+				permanent: false
+			},
+			props: {}
+		};
 	}
 };
 
+export const Callback = () => {};
```
Comment (Krysset) [inline frontend/src/pages/callback.tsx:37]: Empty function?

Comment (Supergamer1337) [reply frontend/src/pages/callback.tsx:37]: Den når aldrig UI:t, men den behöver någon funktion ifall den skulle komma dit

Comment (Supergamer1337) [reply frontend/src/pages/callback.tsx:37]: Det är verkligen bara för att Next.js ska vara glad

Comment (Oscariremma) [reply frontend/src/pages/callback.tsx:37]: We always redirect server side so this is never called but next JS needs it to exist

**Review by Krysset — APPROVED** [review]
LGTM 👍

**Review by GAsplund — APPROVED** [review]
LGTM!

**Pull request merged**